templates/bth/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+
+services:
+  bth:
+    image: nhsuk/bth-wagtail:${DOCKER_IMAGE_TAG}
```

Comment:
This differs from the naming convention expected by the current or proposed CI script.